CWP/papers/HSF-CWP-2017-01_roadmap/authors/address.txt

```diff
@@ -79,6 +79,7 @@ UCincinnati: University of Cincinnati, Cincinnati, OH, USA
 UCIrvine: Department of Physics and Astronomy, University of California Irvine, Irvine, CA, USA
 UCSD: University of California, San Diego, La Jolla, CA, USA
 UDuke: Department of Physics, Duke University, Durham, NC, USA
+UEdinburgh: School of Physics and Astronomy, University of Edinburgh, Edinburgh, United Kingdom
 UGangneung: Gangneung-Wonju National University, South Korea
 UGlasgow: SUPA - School of Physics and Astronomy, University of Glasgow, Glasgow, United Kingdom
 UEDINBURGH: SUPA - School of Physics and Astronomy, University of Edinburgh, Edinburgh, United Kingdom
```

Comment:
I confess I am not checking the full file but see also UEDINBURGH whereas you are adding UEdinburgh. The only difference between the 2 is that the former (same for UGlasgow) contains "SUPA", which is the Scottish Universities Physics Alliance. Maybe @graeme-a-stewart knows whether it is now standard to have both SUPA and these university names together?

Reply:
Rob is in the School of Physics and Astronomy at the University (https://www.ph.ed.ac.uk/people/robert-currie), so it should be UEDINBURGH, with SUPA as part of the name.